playwright/src/main/java/com/microsoft/playwright/junit/Config.java

```diff
@@ -0,0 +1,59 @@
+package com.microsoft.playwright.junit;
+
+import com.microsoft.playwright.APIRequest;
+import com.microsoft.playwright.Browser;
+import com.microsoft.playwright.BrowserType;
+
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Config {
+  private BrowserType.LaunchOptions launchOptions = new BrowserType.LaunchOptions();
+  private Browser.NewContextOptions contextOption = new Browser.NewContextOptions();
+  private APIRequest.NewContextOptions apiRequestOptions = new APIRequest.NewContextOptions();
+  private Map<String, Object> clientValues = new HashMap<>();
+
+  private String browserName = "chromium";
+
+  public String browserName() {
+    return browserName;
+  }
+
+  public void setBrowserName(String browserName) {
+    this.browserName = browserName;
+  }
+
+  public void setHeadless(boolean headless) {
+    launchOptions.setHeadless(headless);
+  }
+
+  public void setBaseURL(String url) {
+    contextOption.setBaseURL(url);
+  }
+
+  public void setStorageStatePath(Path path) {
+    contextOption.setStorageStatePath(path);
```

Comment:
@yury-s which context options do we want the user to be able to set directly vs via providing ContextOptions?

Reply:
>  which context options do we want the user to be able to set directly vs via providing ContextOptions? Do we want to have parity with the nodejs version.

There is no consensus at the moment. It's nice to have a shortcut for the most common ones like `headless`, `viewport` and `baseURL` but we are not sure if we need them for all properties.

> Also in the [node version](https://playwright.dev/docs/test-use-options#more-browser-and-context-options) you can specify certain options directly and also inside of certain objects like launchOptions and contextOptions. Do we want to do this also? If so, which one takes precedent if both are provided and conflict with each other?

 In Node.js all properties can be set directly but some of them have shortcuts too. We want some shortcuts, in Node.js it is programmed so that short form overrides long one. We can do the same in java, I don't think there was a strong reason for that order over the other, we just want the order to be deterministic.